boxsdk/auth/oauth2.py

```diff
@@ -331,20 +331,20 @@ def _oauth_exception(
         Create a BoxOAuthException instance to raise. If the error response is JSON, parse it and include the
         code and message in the exception.
         """
-        exception_kwargs = dict(
-            status=network_response.status_code,
-            url=url,
-            method='POST',
-            network_response=network_response,
-        )
+        exception_kwargs = {
```

Comment:
Why did you have to change the way you initializing dictionary?

Reply:
I didn't have to but it was detected by pylint in Python 3.8 (before pylin for Python 3.6 was fine with that one). It is considered as a good practice now